src/snowflake/cli/plugins/snowpark/snowpark_shared.py

```diff
@@ -74,6 +76,7 @@ def snowpark_package(
             if do_download:
                 log.info("Installing non-Anaconda packages...")
                 should_continue, second_chance_results = package_utils.install_packages(
+                    anaconda,
```

Comment:
This allows us to reduce number of request to Anaconda (not that we do them a lot...)